Heart Disease Prediction/requirements.txt

```diff
@@ -0,0 +1,80 @@
+asgiref @ file:///tmp/build/80754af9/asgiref_1594338739818/work
```

Comment:
The requirements file is overpopulated with unnecessary modules. Clear it and make sure to include only those modules which are used in the script